tests/data-url.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const tmp = require('tmp-sync');
+const dataUrlTransformer = require('../addons/with-web/data-url');
+
+let dir = '';
+
+beforeAll(() => {
```

Comment:
I think we may simplify to just:

```js
const dir = tmp.in();
```

Notice that I've removed the `__dirname` as well.